server.go

```diff
@@ -366,17 +366,16 @@ func (s *Server) Start() {
 	}
 
 	// Read Traces Forever!
-	go func() {
-		defer func() {
-			s.ConsumePanic(recover())
-		}()
-		if s.TraceAddr != nil {
-			// If we ever use multiple readers, pass in the appropriate reuseport option
+	if s.TracingEnabled() {
```

Comment:
I didn't have to do this, but I felt like it was a more reliable way of determining if this should be done. Consolidating the logic to that single method is an improvement imnsho.